src/components/modals/NewMemberModalHeader.jsx

```diff
@@ -5,10 +5,21 @@ import { ReactComponent as Twitter } from "@unioncredit/ui/lib/icons/twitter-fil
 import JSConfetti from "js-confetti";
 import { useEffect, useRef } from "react";
 import { Text, Heading, ButtonRow, Button } from "@unioncredit/ui";
+import getProfileUrl, { generateTwitterLink } from "utils/generateLinks";
+import { useAccount, useNetwork } from "wagmi";
```

Comment:
Nit: Move this to line 1? I try and keep the module imports eg `wagmi`, `ethers` etc seperate from project imports eg `constants`, `hooks` etc.